puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/updateJobs.groovy

```diff
@@ -0,0 +1,39 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Update Jenkins Jobs') {
+            steps {
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: 'master']],
+                    extensions: [[$class: 'CleanCheckout']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/foreman-infra.git']
+                    ]
+                ])
+
+                dir("./ci") {
+                    sh """
+                    virtualenv update_jobs
+                    source update_jobs/bin/activate
+                    pip install jenkins-job-builder
+                    jenkins-jobs --conf ~/jenkins_jobs.ini update --delete-old -r .
```

Comment:
Given shift in idea, how is this being handled now? The credential side of things.